docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/auto-device-selection.rst

```diff
@@ -95,6 +95,10 @@ model and perform the first inference) is reduced when using AUTO. For example:
 
    The longer the process runs, the closer realtime performance will be to that of the best-suited device.
 
+.. note::
+
+   **It is not recommended to test accuracy with AUTO.** Since CPU and GPU(or other target device) have slightly different accuracy numbers, AUTO may have different accuracy numbers in run to run due to different numbers of inferences running on CPU and GPU, especially when testing accuracy with a small number of inputs. Disable CPU acceleration via ``ov::intel_auto::enable_startup_fallback`` can result to run inference on GPU (or target device) only.
```

Comment:
```suggestion
   **Testing accuracy with the AUTO device is not recommended.** Since the CPU and GPU (or other target devices) may produce slightly different accuracy numbers, using AUTO could lead to inconsistent accuracy results from run to run due to a different number of inferences on CPU and GPU. This is particularly true when testing with a small number of inputs. To achieve consistent inference on the GPU (or another target device), you can disable CPU acceleration using ``ov::intel_auto::enable_startup_fallback``.
```

Reply:
Is ``ov::intel_auto::enable_startup_fallback`` should be set to false or just included? in the first case it's: disable CPU acceleration by setting ``ov::intel_auto::enable_startup_fallback`` to false, if just included the text in the note is fine

Reply:
"by setting ov::intel_auto::enable_startup_fallback to false" is more unambiguous.